src/libs/banners/banners.ts

```diff
@@ -69,59 +58,92 @@ export const getBridgeBanners = (
     })
   }
 
-  return activeRoutes
-    .filter(isBridgeTxn)
-    .filter((route) => {
-      if (route.routeStatus === 'failed') return false
-      if (route.routeStatus !== 'ready') return true
-      // If the route is ready to be signed, we should display the banner only if it's not turned into an account op
-      // because when it does get turned into an account op, there will be a different banner for that
-      return !isRouteTurnedIntoAccountOp(route)
-    })
-    .map((r) => {
-      const actions: Action[] = []
+  const filteredRoutes = activeRoutes.filter(isBridgeTxn).filter((route) => {
+    if (route.routeStatus === 'failed') return false
+    if (route.routeStatus !== 'ready') return true
+    return !isRouteTurnedIntoAccountOp(route)
+  })
+
+  const inProgressRoutes = filteredRoutes.filter(
+    (r) => r.routeStatus === 'in-progress' || r.routeStatus === 'waiting-approval-to-resolve'
+  )
 
-      if (r.routeStatus === 'in-progress' || r.routeStatus === 'waiting-approval-to-resolve') {
-        actions.push({
+  const completedRoutes = filteredRoutes.filter((r) => r.routeStatus === 'completed')
+
+  const remainingRoutes = filteredRoutes.filter(
+    (r) => r.routeStatus !== 'in-progress' && r.routeStatus !== 'completed' && r.routeStatus !== 'waiting-approval-to-resolve'
+  )
+
+  const banners: Banner[] = []
+
+  // Handle in-progress transactions grouping
+  if (inProgressRoutes.length > 0) {
+    banners.push({
+      id: 'bridge-in-progress',
+      type: 'info',
+      category: 'bridge-in-progress',
+      title: 'Bridge request in progress',
+      text: `You have ${inProgressRoutes.length} bridge request${
+        inProgressRoutes.length > 1 ? 's' : ''
+      } in progress.`,
+      actions: [
+        {
           label: 'Details',
           actionName: 'open-swap-and-bridge-tab'
-        })
-      }
+        }
+      ]
+    })
+  }
 
-      if (r.routeStatus === 'completed') {
-        actions.push({
+  // Handle completed transactions grouping
+  if (completedRoutes.length > 0) {
+    banners.push({
+      id: 'bridge-completed',
+      type: 'success',
+      category: 'bridge-completed',
+      title: 'Bridge request completed',
+      text: `You have ${completedRoutes.length} completed bridge request${
+        completedRoutes.length > 1 ? 's' : ''
+      }.`,
+      actions: [
+        {
           label: 'Close',
           actionName: 'close-bridge',
-          meta: { activeRouteId: r.activeRouteId }
-        })
-      }
-
-      if (r.routeStatus === 'ready') {
-        const isNextTnxForBridging = r.route.currentUserTxIndex >= 1
+          meta: { activeRouteIds: completedRoutes.map((r) => r.activeRouteId) }
+        }
+      ]
+    })
+  }
 
```

Comment:
"Bridge" in the title can be plural too:
![image](https://github.com/user-attachments/assets/7d43fd9c-9762-4dda-a4d0-03315fe6689d)

Reply:
request*